### PR TITLE
✨ implement wiz parser

### PR DESCRIPTION
**Pull request by manuel-sommer**
see issue #9602

### CODE REVIEW

**Comment by barucijah** [comment]
@manuel-sommer Thanks, appreciate your effort!

**Review by mtesauro — APPROVED** [review]
Approved

File: unittests/scans/wiz/multiple_findings.csv
Comment (cneill) [inline unittests/scans/wiz/multiple_findings.csv:1]: Can you cut this file down a bit? >5MB is overkill here

Comment (manuel-sommer) [reply unittests/scans/wiz/multiple_findings.csv:1]: sure

**Review by cneill — APPROVED** [review]

**Review by grendel513 — APPROVED** [review]

**Review by Maffooch — APPROVED** [review]

**Pull request merged**